libshaderc/include/shaderc.hpp

```diff
@@ -109,21 +109,29 @@ class CompileOptions {
     other.options_ = nullptr;
   }
 
-  // Adds a predefined macro to the compilation options.
+  // Adds a predefined macro to the compilation options. The macro name and
+  // value are passed in with their char* pointers of their strings and their
```

Comment:
Stale bait.  Just omit this and say it behaves the same as `shaderc_compile_options_add_macro_definition`.